inbm-lib/inbm_common_lib/utility.py

```diff
@@ -271,3 +271,63 @@ def validate_file_type(path: list[str]) -> None:
     if os.path.exists(TEMP_EXT_FOLDER):
         shutil.rmtree(TEMP_EXT_FOLDER, ignore_errors=True)
     remove_file_list(extracted_file_list)
+
+
+def get_os_version() -> str:
+    """Get os version from os release file.
+
+    @return value of the VERSION
+    """
+    if os.path.exists(OS_RELEASE_PATH):
+        with open(OS_RELEASE_PATH, 'r') as version_file:
+            content = version_file.read()
+
+        content_dict = parse_os_release(content)
+        for key in content_dict:
```

Comment:
can retrieve key directly with `content_dict.get("VERSION")` -- then check if result is None

Reply:
Updated with this method.